### PR TITLE
init Sentry and capture error strings when logged

### PR DESCRIPTION
**Pull request by piyalbasu**
We can continue to tune our level of error capturing in the future releases. This seems solid for a v1

### CODE REVIEW

File: src/helpers/log.ts [hunk 1]
```diff
@@ -59,6 +60,7 @@ export const log = {
   },
 
   error: ({ title, body = "" }: { title: string; body?: string | object }) => {
+    sentryCaptureMessage(title);
```
Comment (piyalbasu) [inline src/helpers/log.ts:63]: @quietbits has done a great hob making sure thrown errors eventually bubble up to this logging method, so let's just capture everything here

File: src/ducks/sendPayment.ts [hunk 2]
```diff
@@ -28,6 +29,7 @@ export const sendPaymentAction = createAsyncThunk<
         isPubnet: pubnet,
       });
     } catch (error) {
+      sentryCaptureException(error);
```
Comment (piyalbasu) [inline src/ducks/sendPayment.ts:32]: this is the only spot I found where an error would not be caught by the logger

Comment (quietbits) [reply src/ducks/sendPayment.ts:32]: I think it might be because we are logging the error on the UI side. 🤔 Can't think of any other reason.

**Review by quietbits — APPROVED** [review]
🎉

File: src/ducks/sendPayment.ts (hunk 2 above)
Comment (quietbits) [reply src/ducks/sendPayment.ts:32]: Actually, I just tested it and the error wasn't logged. @piyalbasu Could you please add it here? Then we can remove the Sentry from here. It was an oversight on my part for missing it here. 😬

**Pull request merged**